client/src/commons/CheckBox/CustomCheckbox.tsx

```diff
@@ -32,9 +33,11 @@ export default CustomCheckbox;
 
 interface CheckboxElement extends CheckboxProps {
     labelText: string | JSX.Element;
+    'test-id'?: string;
```

Comment:
is this needed

Reply:
Yes, it's so we can add a `test-id` attribute to checkboxes.